x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/evaluate_condition.ts

```diff
@@ -115,7 +115,7 @@ const getData = async (
     if (!nodes.length) return { [UNGROUPED_FACTORY_KEY]: null }; // No Data state
 
     return nodes.reduce((acc, n) => {
-      const nodePathItem = last(n.path) as any;
+      const nodePathItem = last(n.path)!;
```

Comment:
Is this a safe assertion?

Reply:
Yes, the `getNodePath` method used to generate this value always returns an array with at least one item in it.